independent-projects/enforcer-rules/src/main/resources/enforcer-rules/quarkus-banned-dependencies.xml

```diff
@@ -0,0 +1,110 @@
+<enforcer>
+    <rules>
+        <bannedDependencies>
```

Comment:
Really, I don't want us to have duplicates of the rules. It's going to be a nightmare to maintain. Can't you reference the other files in the default one? It doesn't work?

If not, then...

If we can override the Java and Maven rules at the project level when using these rules, I'm fine with having them in the rule file and having only one of them.
If not, let's have separate files but not the file duplicating everything.
Using XSLT is really not natural for most Java developers even if I know the Camel folks are fond of it :).

Reply:
I've got rid of `quarkus-default.xml` and made them separate